src/auto/terminal.rs

```diff
@@ -1,1024 +1,2015 @@
+
+    fn set_audible_bell(&self, is_audible: bool);
+
+    fn set_backspace_binding(&self, binding: EraseBinding);
+
+    #[cfg(any(feature = "v0_52", feature = "dox"))]
+    fn set_bold_is_bright(&self, bold_is_bright: bool);
+
+    #[cfg(any(feature = "v0_52", feature = "dox"))]
+    fn set_cell_height_scale(&self, scale: f64);
+
+    #[cfg(any(feature = "v0_52", feature = "dox"))]
+    fn set_cell_width_scale(&self, scale: f64);
+
+    fn set_cjk_ambiguous_width(&self, width: i32);
+
+    #[cfg(any(feature = "v0_52", feature = "dox"))]
+    fn set_clear_background(&self, setting: bool);
+
+    fn set_color_background(&self, background: &gdk::RGBA);
+
+    fn set_color_bold(&self, bold: Option<&gdk::RGBA>);
+
+    fn set_color_cursor(&self, cursor_background: Option<&gdk::RGBA>);
+
+    #[cfg(any(feature = "v0_44", feature = "dox"))]
+    fn set_color_cursor_foreground(&self, cursor_foreground: Option<&gdk::RGBA>);
+
+    fn set_color_foreground(&self, foreground: &gdk::RGBA);
+
+    fn set_color_highlight(&self, highlight_background: Option<&gdk::RGBA>);
+
+    fn set_color_highlight_foreground(&self, highlight_foreground: Option<&gdk::RGBA>);
+
+    fn set_cursor_blink_mode(&self, mode: CursorBlinkMode);
+
+    fn set_cursor_shape(&self, shape: CursorShape);
+
+    fn set_default_colors(&self);
+
+    fn set_delete_binding(&self, binding: EraseBinding);
+
+    #[cfg_attr(feature = "v0_54", deprecated)]
+    fn set_encoding(&self, codeset: Option<&str>) -> Result<(), Error>;
+
+    fn set_font(&self, font_desc: Option<&pango::FontDescription>);
+
+    fn set_font_scale(&self, scale: f64);
+
+    //#[cfg_attr(feature = "v0_52", deprecated)]
+    //fn set_geometry_hints_for_window(&self, window: /*Ignored*/&gtk::Window);
+
+    fn set_input_enabled(&self, enabled: bool);
+
+    fn set_mouse_autohide(&self, setting: bool);
+
+    //fn set_pty(&self, pty: /*Ignored*/Option<&Pty>);
+
+    fn set_rewrap_on_resize(&self, rewrap: bool);
+
+    fn set_scroll_on_keystroke(&self, scroll: bool);
+
+    fn set_scroll_on_output(&self, scroll: bool);
+
+    fn set_scrollback_lines(&self, lines: libc::c_long);
+
+    fn set_size(&self, columns: libc::c_long, rows: libc::c_long);
+
+    //#[cfg(any(feature = "v0_52", feature = "dox"))]
+    //fn set_text_blink_mode(&self, text_blink_mode: /*Ignored*/TextBlinkMode);
+
+    #[cfg(any(feature = "v0_40", feature = "dox"))]
+    fn set_word_char_exceptions(&self, exceptions: &str);
+
+    //#[cfg(any(feature = "v0_48", feature = "dox"))]
+    //fn spawn_async(&self, pty_flags: /*Ignored*/PtyFlags, working_directory: Option<&str>, argv: &[&std::path::Path], envv: &[&std::path::Path], spawn_flags_: /*Ignored*/glib::SpawnFlags, child_setup: Option<Box<dyn Fn() + 'static>>, timeout: i32, cancellable: /*Ignored*/Option<&gio::Cancellable>, callback: /*Unimplemented*/Fn(&Terminal, /*Ignored*/glib::Pid, &Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>);
```

Comment:
I think you will need this method as well for your project.
Don't you?

Reply:
Yup, I'll add it once I can generate "Vte.Pty"